src/sentry/ingest/outcomes_consumer.py

```diff
@@ -91,41 +93,20 @@ def process_message(self, message):
 
         # remember that we sent the signal just in case the processor dies before
         mark_signal_sent(project_id=project_id, event_id=event_id)
+        return True
```

Comment:
We probably don't need this now since we don't do advanced batching (yet), so each batch will be basically ignored.

Reply:
Yes this should go. I was unsure about the contract of AbstractBatchWorker

Reply:
Done!